src/coredump/coredumpctl.c

```diff
@@ -575,7 +575,7 @@ static int dump_list(sd_journal *j) {
         return 0;
 }
 
-static int save_core(sd_journal *j, int fd, char **path, bool *unlink_temp) {
+static int save_core(sd_journal *j, int fd, bool refuse_tty, char **path, bool *unlink_temp) {
```

Comment:
OK, I'll have another look at restucturing the code… Last time I tried was before some of the cleanups, so maybe it will be easier now. Current version feels weird for me too.